configuration/configmgr/configmgrlib/mod_templates/EnvModTemplate.cpp

```diff
@@ -0,0 +1,341 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2018 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+#include "EnvModTemplate.hpp"
+#include "../../RapidJSON/include/rapidjson/error/en.h"
```

Comment:
Had a few issues getting the include to work right (cmake rookie). I believe I have it worked out now. The include path is now set so that rapid json headers are referenced as "rapidjson/header.h